client/src/components/main.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import { Toolbar } from "primereact/toolbar";
+import { Button } from "primereact/button";
+import { InputText } from "primereact/inputtext";
+import { Dropdown } from "primereact/dropdown";
+import { Sidebar } from "primereact/sidebar";
+import { useNavigate, Routes, Route } from "react-router-dom";
+import CartPopUp from "./cartPopUp";
+import Item from "./item/item";
+import Order from "./order/order";
+import { AutoComplete } from "primereact/autocomplete";
+import { itemsStore } from "../store/itemsStore";
+import { ordersStore } from "../store/ordersStore";
+
+import "primeicons/primeicons.css";
+import "primereact/resources/themes/saga-blue/theme.css";
+import "primereact/resources/primereact.css";
+import "primeflex/primeflex.css";
+import "../comCss/main.css";
+
+export default function Main() {
+  const [activeSidebar, setActiveSidebar] = useState(false);
+  const [screanFlag, setScreanFlag] = useState(false);
+  const compNavigate = useNavigate();
+
+  const navigateToItems = () => {
+    setScreanFlag(false);
+    compNavigate("/items");
+  };
+
+  const navigateToOrders = () => {
+    setScreanFlag(true);
+    compNavigate("/orders");
+  };
+
+  const citySelectItems = [
+    { label: "FRUITS", value: "FRUITS" },
+    { label: "VEGETABLES", value: "VEGETABLES" },
+    { label: "MEAT", value: "MEAT" },
+    { label: "DAIRY", value: "DAIRY" },
+    { label: "*", value: "ALL_ITEMS" },
+  ];
+
+  const leftToolBar = (
+    <React.Fragment>
+      <CartPopUp></CartPopUp>
+      <i>הסל שלי</i>
+    </React.Fragment>
+  );
+
+  const rightToolBar = (
+    <React.Fragment>
+      <div
+        className="col-12 md:col-4 input-order-item-by-id "
+        style={{ visibility: screanFlag ? "visible" : "hidden" }}
+      >
+        <div className="p-inputgroup">
+          <InputText placeholder="חיפוש שם לפי מזהה"
+          onChange={(e) => {ordersStore.searchOrderById(e.target.value)}} />
+        </div>
+      </div>
+      <div
+        className="div-item-search"
+        style={{ visibility: screanFlag ? "hidden" : "visible" }}
```

Comment:
add css class

Reply:
I use the screanFlag state...  can I move it to css?